src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyCompactCtors.java

```diff
@@ -63,6 +63,18 @@ record MyRecord6() {
     }
 
     record TestRecord7() {
-        public TestRecord7 {} // ok
+        public TestRecord7{} // violation, no WS before '{'
     }
```

Comment:
Please keep existing input java code unchanged. If need more case create new method/classes.
We can change only content of trailing comments

Reply:
I moved them out of the main class. I guess you were asking for this and not a new input file.